nle/__init__.py

```diff
@@ -1,6 +1,6 @@
 # Copyright (c) Facebook, Inc. and its affiliates.
 #
 # This is needed to register envs
-import nle.env
+# import nle.env
```

Comment:
should just delete if not needed

Reply:
Right :)

This was just a leftover-comment from when nothing worked on this branch. Will revert to what it was originally.